### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by AfubeAngel**
# Pull Request Template

## What
Updated the README.md and CONTRIBUTING.md files to enhance clarity and guidance for new contributors.

## How
Revised sections for installation and contribution guidelines to improve usability and understanding. Added detailed instructions on setting up the development environment and contributing effectively.

## Additional Notes
These updates aim to streamline the onboarding process for new contributors and improve overall project documentation.

issue #94

### CODE REVIEW

**Comment by AfubeAngel** [comment]
@manuelalferez Please assist with the authorization for the deployment. I updated the CONTRIBUTING.md as directed and included the changes you made in the README.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,70 @@
-# Contributing
+# Contributing to Flowinance
 
-When contributing to this repository, please first discuss the change you wish to make [**creating an issue**](https://github.com/manuelalferez/flowinance/issues).
+We’re so glad you’re interested in contributing to our **open-source project**! Before getting started, please take a moment to read our guidelines to ensure a smooth and efficient contribution process.
```
Comment (manuelalferez) [inline CONTRIBUTING.md:3]:
```suggestion
We are glad that you are interested in contributing to our open source project!
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,10 +1,70 @@
...
+
+## 💬 Have feedback or questions? Join the conversation!
+
+We encourage you to visit our [Flowinance Discussions](https://github.com/manuelalferez/flowinance/discussions) for general questions, feature suggestions, or to engage with the community. It's the best place to share ideas and collaborate!
+
+## Ask me 🤙
+
+You can always contact me via [Telegram](https://t.me/manuelalferez) if you want to talk more about the project 😊
+
+Thank you for your contributions! Together, we can make Flowinance even better 🎉
```
Comment (manuelalferez) [inline CONTRIBUTING.md:70]:
```suggestion
```

Comment (manuelalferez) [reply CONTRIBUTING.md:70]: We make contributors read a lot: we will respect everyone's time.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,10 +1,70 @@
...
 
-## Pull Request Process
+## 📝 Contribution Guidelines
+1. Discuss Changes Before Starting
+Please first discuss the change you wish to make [**creating an issue**](https://github.com/manuelalferez/flowinance/issues).
 
-1. Make sure the change runs successfully locally
-2. Consider if the change affects `README.md`
-3. In your commit message's subject (50 char length at max) you should succinctly describe what changes this commit introduces
-4. A commit shouldn't take more than 100 lines of code. You will write your contribution faster and the person who will review your code will appreciate it.
```
Comment (manuelalferez) [inline CONTRIBUTING.md:10]: Please, keep this.

File: CONTRIBUTING.md (hunk 2 above)
Comment (AfubeAngel) [reply CONTRIBUTING.md:70]: @manuelalferez Apologies. I wanted to break it down for first-time contributors, I guess it isn't necessary. Apologies again

**Comment by AfubeAngel** [comment]
@manuelalferez Is it alright with you if I  close this pull request and open a new one considering all the corrections? As you mentioned, the README.md is fine. So I added the table of contents for faster navigation and updated the prerequisites to take supabase into account. I apologize for the duplication. The primary changes were done in the CONTRIBUTING.md which I had to write with fresh contributors or beginners in mind but since it is too long; I would shorten it according to your directions.

Or should I go ahead and make the changes on this pull request? Thank you

**Comment by manuelalferez** [comment]
@AfubeAngel It is better to keep everything in the same PR, this way the whole history of the improvement is kept in a single point and not in multiple places. But as it is easier for you, if you see that it is going to be too much, open a new one 😊

**Comment by AfubeAngel** [comment]
@manuelalferez I'll just push on this PR.

Please is this shorter version acceptable for the contributing.md?
image

**Comment by manuelalferez** [comment]
@AfubeAngel The guideline is already an explanation of how to contribute, the how to contribute only explains how to install the project: that information is already in the README. maybe you can change it to something like: Installation, 
“Follow the instructions in the readme to install and run the project” and add a link

**Comment by manuelalferez** [comment]
@AfubeAngel great job!! Much better. Ready to merge

**Pull request merged**